.github/workflows/compatibility-report.yml

```diff
@@ -54,17 +54,59 @@ jobs:
           path: compatibility.md
 
       # Generate the new PR to update the doc in the repo.
-      - run: |
-          git config user.name $GITHUB_ACTOR
-          git config user.email gh-actions-${GITHUB_ACTOR}@github.com
       - run: |
           rm -f status.log
       - uses: peter-evans/create-pull-request@6d6857d36972b65feb161a90e484f2984215f83e
         with:
+          token: ${{ secrets.NODE_AGENT_GH_TOKEN || secrets.GITHUB_TOKEN }}
           title: "docs: Updated compatibility report"
           commit-message: "docs: Updated compatibility report"
           branch: "compatibility-report/auto-update"
           delete-branch: true
           base: main
           labels: "documentation"
 
+  docs:
```

Comment:
I'm not yet sure this job will do what we actually want, but I can't run it from a branch because this PR is originating from a fork. I think it will just be easier to merge it as-is, and then I can do some manual testing and submit another PR if things are not correct.

Reply:
that's fair